charts/metering-ansible-operator/templates/crds/reportquery.crd.yaml

```diff
@@ -23,3 +23,104 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+
+  validation:
+    openAPIV3Schema:
+      type: object
+      required:
+      - spec
+      properties:
+        spec:
+          type: object
+          required:
+          - columns
+          - query
+          properties:
+            columns:
+              type: array
+              minItems: 1
+              items:
+                type: object
+                required:
+                - name
+                - type
+                properties:
+                  name:
+                    type: string
+                    minLength: 1
+                  type:
+                    type: string
+                    enum:
```

Comment:
None of our existing queries use uppercase, so we need to update those, or add lowercase support for these. Given how many queries we have, it may be easier to just relax the validation to support either all uppercase, or all lowercase than to update all our queries.

Now that we have the list of columns right here, it should be really easy to add the lowercase versions.

- copy the list, paste it below the current list (duplicating the items)
- select the newly pasted items, and lowercase the selected text
     - most editors make this easy: https://stackoverflow.com/questions/38790154/keyboard-shortcut-to-convert-selection-to-uppercase-or-lowercase-in-the-atom-e